Raspi/Test/mock_wireless_module.py

```diff
@@ -177,14 +196,15 @@ def start_modules(args):
     # TODO: Add posibility to make modules by importing a file or generating
     # random modules. The modules should not be hard coded to this script.
 
+    print()
     if args.id is None:
-        for i in range(1, 4):
+        for i in range(1, 5):
```

Comment:
Should we create a global variable to keep track of the number of modules available?